pyomo/contrib/parmest/tests/test_parmest.py

```diff
@@ -231,6 +240,29 @@ def test_theta_est(self):
         self.assertAlmostEqual(thetavals['k2'], 5.0/3.0, places=4) 
         self.assertAlmostEqual(thetavals['k3'], 1.0/6000.0, places=7) 
         
-
+@unittest.skipIf(not parmest.parmest_available,
+                 "Cannot test parmest: required dependencies are missing")
+@unittest.skipIf(not graphics.imports_available,
+                 "parmest.graphics imports are unavailable")
+@unittest.skip("Temporarily disabling to track down testing failures")
+class parmest_graphics(unittest.TestCase):
+    
+    def setUp(self):
+        self.A = pd.DataFrame(np.random.randint(0,100,size=(100,4)), columns=list('ABCD'))
+        self.B = pd.DataFrame(np.random.randint(0,100,size=(100,4)), columns=list('ABCD'))
+        
+    def test_pairwise_plot(self):
+        filename=os.path.abspath(os.path.join(testdir, 'simple_pairwise_plot.png'))
```

Comment:
Seems like these tests will create 3 png files that are not deleted after testing is completed. Was there a reason these tests were set up to save the figures instead of just setting the filename to `None`?

Reply:
@blnicho setting the filename to None is fine. Thanks for working through these updates.